src/App.js

```diff
@@ -1,20 +1,38 @@
 import React, { Component } from 'react';
+import SearchBar from './components/SearchBar';
 
 class App extends Component {
+  
   render() {
+    const { contentStyle, introStyle } = styles;
+
     return (
-      <h1 style={styles.introStyle}>This is a Youtube Clone site and is under development.</h1>
+      <div style={contentStyle}>
+        <div style={introStyle}>
+          <h1>This is a Youtube Clone site and is under development.</h1>
+        </div>
+
+        <SearchBar />
+      
+      </div>
     );
   }
 }
 
 const styles = {
-  introStyle: {
+  contentStyle: {
```

Comment:
Can you use a grid system of material-ui and realize a similar design?
https://material-ui.com/layout/grid/
in my opinion, coding direct css is to keep to a minimum as far as possible.